### PR TITLE
[ADD] purchase_order_line_sequence v10

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
Purchase order lines with sequence number
=========================================

Provides a new field on the purchase order, which allows to manage the order of its purchase order lines.

### CODE REVIEW

**Comment by serpentcs-dev1** [comment]
@aheficent Could you please review?

File: purchase_order_line_sequence/README.rst [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
     :alt: License: AGPL-3
 
+=========================================
 Purchase order lines with sequence number
 =========================================
 
-Provide a new field on the purchase order form, allowing to manage the order of the lines
-order.
+Provides a new field on the purchase order, which allows to manage the order of its purchase order lines.
+
```
Comment (AaronHForgeFlow) [inline purchase_order_line_sequence/README.rst]: Can you mention that the sequence is propagated to the invoice lines?

File: purchase_order_line_sequence/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Purchase order lines with sequence number',
+    'summary': '',
```
Comment (AaronHForgeFlow) [inline purchase_order_line_sequence/__manifest__.py]: Please add a summary

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Functional test ok, the sequence number, the copy, the previous PO sequence and the reports. Only some minor suggestions.

File: purchase_order_line_sequence/__manifest__.py (hunk 2 above)
Comment (serpentcs-dev1) [reply purchase_order_line_sequence/__manifest__.py]: Oh, Yes!!  :grimacing:

File: purchase_order_line_sequence/README.rst (hunk 1 above)
Comment (serpentcs-dev1) [reply purchase_order_line_sequence/README.rst]: Sure, it propagates to invoice lines and the stock moves.

I'll add it!

**Review by AaronHForgeFlow — COMMENTED** [review]
The code LGTM, just one thing.

File: purchase_order_line_sequence/__init__.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import models
+from . import tests
```
Comment (AaronHForgeFlow) [inline purchase_order_line_sequence/__init__.py]: I think this is not needed.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Thank you for attending the comments

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
IMHO the hook's file should be renamed to hooks.py

**Comment by AaronHForgeFlow** [comment]
No comments on my last review? :cry:

**Comment by serpentcs-dev1** [comment]
Done :smile:

File: purchase_order_line_sequence/tests/test_po_lines_sequence.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        self.po.button_confirm()
+
+        move1 = self.env['stock.move'].\
+            search([('purchase_line_id', '=', self.po.order_line[0].id)])
+        move2 = self.env['stock.move'].\
+            search([('purchase_line_id', '=', self.po.order_line[1].id)])
+
+        self.assertEqual(self.po.order_line[0].sequence,
+                         move1.sequence,
+                         'The Sequence of the Purchase Order Lines does not '
```
Comment (AaronHForgeFlow) [inline purchase_order_line_sequence/tests/test_po_lines_sequence.py:63]: This test is failing :disappointed:

**Pull request closed without merging**